### ISSUE TITLE
AFT v1.6.5 fails aft_account_provisioning_framework_account_metadata_ssm with >10 custom-fields SSM parameters

### ISSUE THREAD

**Issue by fjromerom** [aws-ia/terraform-aws-control_tower_account_factory#262]
**AFT Version:**
1.6.5

**Bug Description**
I've upgraded AFT from v1.6.4 to v1.6.5 and we are experiencing issues when updating account-request file of an account with more than 10 custom-fields. The Step Function aft-account-provisioning-framework fails at Lambda function  aft_account_provisioning_framework_account_metadata_ssm.
However, creation of a new AWS account succeeds.

**To Reproduce**
1. Update account-request file of an account with more 10 custom-fields
2. Step Function aft-account-provisioning-framework will fail at Lamba function aft_account_provisioning_framework_account_metadata_ssm with the error below.

**Expected behavior**
Step Function aft-account-provisioning-framework should success with no errors.

**Related Logs**
Lambda Function aft_account_provisioning_framework_account_metadata_ssm error message:

```
{
  "resourceType": "lambda",
  "resource": "invoke",
  "error": "ClientError",
  "cause": {
    "errorMessage": "An error occurred (ValidationException) when calling the DeleteParameters operation: 1 validation error detected: Value '[...]' at 'names' failed to satisfy constraint: Member must have length less than or equal to 10.",
    "errorType": "ClientError",
    "stackTrace": [
      "  File \"/var/task/aft_account_provisioning_framework_account_metadata_ssm.py\", line 70, in lambda_handler\n    delete_ssm_parameters(target_account_session, params_to_remove)\n",
      "  File \"/opt/python/lib/python3.8/site-packages/aft_common/account_provisioning_framework.py\", line 315, in delete_ssm_parameters\n    response = client.delete_parameters(Names=parameters)\n",
      "  File \"/opt/python/lib/python3.8/site-packages/botocore/client.py\", line 508, in _api_call\n    return self._make_api_call(operation_name, kwargs)\n",
      "  File \"/opt/python/lib/python3.8/site-packages/botocore/client.py\", line 915, in _make_api_call\n    raise error_class(parsed_response, operation_name)\n"
    ]
  }
}
```

**Additional context**
It's weird the changelog of the v1.6.5 says this issue is resolved while we did not experience this issue before and started to face this issue with this version.

Thanks,
Francisco

**Comment by snebhu3** [comment]
@fjromerom thank you for reporting this issue. 
We are looking into it.

**Comment by stumins** [comment]
@fjromerom We've just released [1.6.6](https://github.com/aws-ia/terraform-aws-control_tower_account_factory) to resolve this issue.

Thanks for the report!
